StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheet+PaymentMethodAvailability.swift

```diff
@@ -8,6 +8,9 @@ import Foundation
 
 extension CustomerSheet {
     static let supportedPaymentMethods: [STPPaymentMethodType] = [.card, .USBankAccount, .SEPADebit]
+    static func paymentMethodIsSupported(_ paymentMethod: STPPaymentMethodType) -> Bool {
+        return supportedPaymentMethods.contains(where: { $0 == paymentMethod })
+    }
```

Comment:
I think this can just be inlined as `supportedPaymentMethods.contains(paymentMethod) `, the helper is overkill